BunqSdk.Tests/Model/Generated/Endpoint/RequestInquiryTest.cs

```diff
@@ -16,16 +16,16 @@ public class RequestInquiryTest : BunqSdkTestBase
         /// <summary>
         /// Config values.
         /// </summary>
-        private const string AMOUNT_EUR = "0.01";
-        private const string FIELD_CURRENCY = "EUR";
-        private const string FIELD_PAYMENT_DESCRIPTION = "C# test Payment";
-        private const string FIELD_STATUS = "ACCEPTED";
-        private const int INDEX_FIRST = 0;
+        private const string Amount = "0.01";
+        private const string Currency = "EUR";
```

Comment:
Correct to remove `Field` as these are not field constants, but please rename to `PaymentCurrencyEur`.